airbyte-integrations/bases/base-normalization/dbt-project-template/dbt_project.yml

```diff
@@ -29,14 +29,20 @@ clean-targets:         # directories to be removed by `dbt clean`
 
 quoting:
   database: true
-  schema: true
+  schema: false
```

Comment:
why?

Reply:
I will add the same comment:
> Temporarily disabling the behavior of the ExtendedNameTransformer, see (issue #1785)

Since we don't use Extended Names for tables and schemas anymore but replace all special characters with `_` then DBT shouldn't be using quoting when querying them otherwise we run into case-sensitive and quoting issues...

The destination may produce schema/table names that are case-insensitive and dbt would be case-sensitive (with quoting)